istio/README.md

```diff
@@ -112,6 +112,112 @@ See [service_checks.json][16] for a list of service checks provided by this inte
 
 ## Troubleshooting
 
+### Using Openmetrics Integration
+
+If Istio proxy sidecar injection is enabled, monitoring other Prometheus metrics via the [Openmetrics integration][20] can result in high custom metrics usage and duplicated metric collection.
+
+To ensure that your Openmetrics configuration is not redundantly collecting metrics, either:
+
+1. Use specific metric matching in the `metrics` configuration option
+2. If you use `*` value for `metrics`, then at minimum, consider including the following optional options to exclude metrics already collected by istio and Envoy inegrations.
```

Comment:
```suggestion
2. If using the wildcard `*` value for `metrics`, then consider including the following optional options in the Istio or Envoy `conf.yaml` to exclude metrics already collected.
```

Reply:
the second half of your suggestion is incorrect. The recommendations below are for the openmetrics configuration... you're not suppose to exclude metrics from Istio or Envoy conf.yaml.

Reply:
Good catch, updated comment to be openmetrics